ui/pages/home/home.container.js

```diff
@@ -86,9 +87,14 @@ const mapStateToProps = (state) => {
   const isPopup = envType === ENVIRONMENT_TYPE_POPUP;
   const isNotification = envType === ENVIRONMENT_TYPE_NOTIFICATION;
 
-  const firstPermissionsRequest = getFirstPermissionRequest(state);
-  const firstPermissionsRequestId =
-    firstPermissionsRequest?.metadata.id || null;
+  let firstPermissionsRequest, firstPermissionsRequestId;
+  firstPermissionsRequest = getFirstPermissionRequest(state);
+  if (firstPermissionsRequest) {
+    firstPermissionsRequestId = firstPermissionsRequest?.metadata.id || null;
+  } else {
+    firstPermissionsRequest = getFirstSnapUpdateRequest(state);
```

Comment:
What is the goal of this? 🤔

Reply:
The home component grabs whatever pending approvals there are and sends them to the `PermissionConnect` component, in the current case, `firstPermissionRequest` would not pick up the pending approval because it's not checking for update requests.